safedelete/models.py

```diff
@@ -160,6 +160,22 @@ def delete(self, force_policy=None, **kwargs):
             # soft-delete the object
             self.delete(force_policy=SOFT_DELETE, **kwargs)
 
+    @staticmethod
+    def has_unique_fields(model):
+        """Checks if one of the fields of this model has a unique constraint set (unique=True)
+
+        Args:
+            model: Model instance to check
+        """
+        for field in model._meta.fields:
+            if field._unique:
+                return True
+        return False
+
+    def get_delete_policy(self):
```

Comment:
Delete the getter, `_safedelete_policy` is a private field and does not need a getter (definitely not in Python).

Reply:
done.